### DISCUSSION TITLE
emit event not working after update to v2.6.7

### DISCUSSION THREAD

**Discussion by Shadecute** [livewire/livewire#3968] [1. Help]
My last version was before the update v2.4.2 
I used this code to display notifications messages,
in livewire component: 
`        $this->emit('toast', 'success',
            'test ok');
`

js:
```
const Toast = Swal.mixin({
    toast: true,
    position: 'center',
    showConfirmButton: false,
    timer: 4000,
    timerProgressBar: true,
    customClass: {
        title: 'persian-font',
    },
    didOpen: (toast) => {
        toast.addEventListener('mouseenter', Swal.stopTimer)
        toast.addEventListener('mouseleave', Swal.resumeTimer)
    }
})


document.addEventListener('livewire:load', () => {
    Livewire.on('toast', (type, message) => {
        Toast.fire({
            icon: type,
            title: message
        })

    })
})

```

And it works well , But after updating to the latest version, this code no longer works,
Also, I do not see any error in the console that I can fix, it just does not work anymore.

**Comment by joshhanley** [comment]
@Shadecute please share your component class and blade view so we can see where and how you are using `emit`.

Thanks!
